### ISSUE TITLE
tag for v20.10.25 is missing

### ISSUE THREAD

**Issue by mrueg** [docker/cli#4294]
### Description

docker 20.10.25 was released https://github.com/moby/moby/releases/tag/v20.10.25 but https://github.com/docker/cli/releases/tag/v20.10.25 wasn't tagged.

### Reproduce

Check https://github.com/docker/cli/releases/tag/v20.10.25

### docker version

```bash
-
```

### docker info

```bash
-
```

**Comment by thaJeztah** [comment]
@corhere yup, gathered that; do you know the commit you used that should be tagged? Then someone with permissions can tag it

**Comment by corhere** [comment]
@thaJeztah b82b9f3a0e763304a250531cb9350aa6d93723c9

**Comment by neersighted** [comment]
I just pushed a [v20.10.25 tag](https://github.com/docker/cli/releases/tag/v20.10.25).

**Comment by kikitux** [comment]
Hello, will be a binary version for docker 20.10.25? the upgrade to go is due a security CVE in golang

**Comment by thaJeztah** [comment]
There's no plans to do further releases of `docker-ce` for the 20.10 branch. Mirantis does have a commercially supported MCR (Mirantis Container Runtime); which has LTS support (https://docs.mirantis.com/mcr/20.10/overview.html), which is based on the same code-base (with additional patches), and of course you can build from source.

If you don't need a commercially supported version, I'd recommend upgrading to the currently maintained version (24.0), but of course verify the upgraded version before rolling out to production.

**Comment by julianocosta89** [comment]
Hello all,
I'd like to revive this thread a bit 😅

@thaJeztah mentioned that:
>There's no plans to do further releases of docker-ce for the 20.10 branch. Mirantis does have a commercially supported MCR (Mirantis Container Runtime); which has LTS support (https://docs.mirantis.com/mcr/20.10/overview.html), which is based on the same code-base (with additional patches), and of course you can build from source.

This sounds fair, but I wonder what is the point of keeping releasing new tags on this repo, if those will never be shipped to https://download.docker.com/linux/static/stable/x86_64/ nor https://download.docker.com/linux/static/stable/aarch64/?

IMHO if the tag was released, that should be shipped.

I understand the pain of LTS, so I'd personally prefer if the fixes were just shipped on the supported versions, and if vendors wanted to backport that, then they can do on their own tags (as Mirantis already does - currently they have 20.10.17 as latest v20 tag).

**Comment by neersighted** [comment]
The reason for tags is because this repository is currently a satellite of moby/moby; every engine release has a corresponding CLI release, even though this is technically a Docker Inc. OSS project and not a part of the Moby project.

I don't think there's any harm in tagging here (which is why we did it); and I'm working on some documentation for moby/moby clarifying the upstream/downstream relationship.

This repository might take more time as it's a bit ambiguous to us as well, but unless some harm reveals itself, I think we'll continue to tag releases, even if there never is a Docker CE release -- it will help those trying to build on the 20.10/23.0 branches, and doesn't cost us anything.
